hls-test-utils/src/Test/Hls.hs

```diff
@@ -323,8 +326,14 @@ mkPluginTestDescriptor'
 mkPluginTestDescriptor' pluginDesc plId _recorder = IdePlugins [pluginDesc plId]
 
 -- | Initialise a recorder that can be instructed to write to stderr by
--- setting the environment variable "HLS_TEST_PLUGIN_LOG_STDERR=1" before
--- running the tests.
+-- setting the environment variable "HLS_TEST_PLUGIN_LOG_STDERR|LSP_TEST_LOG_STDERR
+-- |LSP_TEST_LOG_STDERR|HLS_TEST_LOG_STDERR=1" before running the tests.
+--
+-- Because "LSP_TEST_LOG_STDERR" has been used before,
+-- (thus, backwards compatibility) and "HLS_TEST_HARNESS_STDERR" because it
+-- uses a more descriptive name to be used with "HLS_TEST_HARNESS_NO_TESTDIR_CLEANUP".
```

Comment:
This is not the intention, a plugin test recorder should not log on `HLS_TEST_HARNESS_STDERR`.

`HLS_TEST_HARNESS_STDERR` should log information about our test helpers, such as `runSessionWithServerInTmpDirCont`, it shouldn't enable plugin logging. Same goes for `HLS_TEST_HARNESS_NO_TESTDIR_CLEANUP`, that environment variable shouldn't be used for logging at all, it doesn't remove a `VirtualFileTree` after the test is run.

Reply:
so `HLS_TEST_LOG_STDERR` enable both the helper and plugin, while `HLS_TEST_HARNESS_STDERR ` and `HLS_TEST_PLUGIN_LOG_STDERR ` enable either.

Reply:
Yes, that's exactly what I meant, thanks!